recipes/rotary-embedding-torch/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "rotary-embedding-torch" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rotary-embedding-torch-{{ version }}.tar.gz
+  sha256: 45db29b19bd7025f09d202752c26bf6921b05d8b5a977cfcdc625ce96ddf2b5c
+
+build:
+  skip: true  # [win]
```

This should be a `noarch` package. Failing CI on Windows is OK as `pytorch` is not available there.
```suggestion
     noarch: python
```